plumbum/commands/processes.py

```diff
@@ -354,11 +354,8 @@ def iter_lines(proc,
 
     assert mode in (BY_POSITION, BY_TYPE)
 
-    encoding = getattr(proc, "custom_encoding", None)
-    if encoding:
-        decode = lambda s: s.decode(encoding).rstrip()
-    else:
-        decode = lambda s: s
+    encoding = getattr(proc, "custom_encoding", None) or 'utf-8'
```

Comment:
```suggestion
    encoding = getattr(proc, "custom_encoding", 'utf-8')
```

Am I missing something or is this the same?

Reply:
it's not the same in case `proc.custom_encoding` is explicitly `None`